keyboards/xbows/ranger/readme.md

```diff
@@ -0,0 +1,20 @@
+# Ranger
+![Ranger](https://img.alicdn.com/i3/16731204/O1CN015gRK1R1KlTSHEuHDo_!!16731204.jpg)
+
+Ranger, A 87 keys keyboard with RGB backlight.
+
+* Keyboard Maintainer: X-BOWS
+* Hardware Supported: X-BOWS Ranger
+* Hardware Availability: [X-BOWS](https://x-bows.com/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?

Reply:
Enter the bootloader in 3 ways:
* Software reset on Fn + ESC
* Bootmagic reset: hold down the top left key (usually escape) and plugin the keyboard
* Physical reset button: There is a button on the back of the PCB, please press it briefly.

Flash new firmware in 2 ways:
* Use the QMK toolbox：https://github.com/qmk/qmk_toolbox/releases
* Use the QMK MSYS：https://msys.qmk.fm/

Reply:
> Could you add instructions on how to enter the bootloader, to flash new firmware?

I added some instructions. Do I need to make any other changes?